app/views/devise/passwords/edit.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="centered-form">
+  <h2>Change your password</h2>
+
+  <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put, class: "needs-validation", novalidate: true }) do |f| %>
+    <% if resource.errors.any? %>
+      <div class="alert alert-danger">
+        <h4 class="alert-heading"><%= pluralize(resource.errors.count, "error") %> prohibited this user from being saved:</h4>
+        <ul>
+          <% resource.errors.full_messages.each do |msg| %>
+            <li><%= msg %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    
+    <%= f.hidden_field :reset_password_token %>
+
+    <div class="mb-3">
+      <%= f.label :password, "New password", class: "form-label" %>
+      <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %> characters minimum)</em><br />
+      <% end %>
+      <%= f.password_field :password, autofocus: true, autocomplete: "new-password", class: "form-control", required: true %>
+      <div class="invalid-feedback">
```

Comment:
```suggestion
      <div class="invalid-feedback">
```

is this needed? does it even show up?
perhaps consider removing all of the "invalid-feedback" divs if its not necessary